crates/revm/Cargo.toml

```diff
@@ -17,6 +17,7 @@ auto_impl = { version = "1.0", default-features = false }
 
 # Optional
 serde = { version = "1.0", features = ["derive", "rc"], optional = true }
+serde_json = { version = "1.0", features = ["preserve_order"] }
```

Comment:
```suggestion
serde_json = { version = "1.0", features = ["preserve_order"], optional = true }
```

Reply:
and add serde_jsoin in serde features.

Reply:
how? serde_json is not in serde features

Reply:
`serde = ["dep:serde", "revm-interpreter/serde"]`
to
`serde = ["dep:serde", "dep:serde_json", "revm-interpreter/serde"]`